eve/main.yml

```diff
@@ -0,0 +1,91 @@
+version: "0.2"
+
+branches:
+  default:
+    stage: "pre-merge"
+
+stages:
+  pre-merge:
```

Comment:
@Zempashi  Is there any reason why you're not directly calling the `single-node` stage?

Reply:
To let other people adding tests that run in parallel..

Reply:
yeah, make sense!